src/useLocalStorage.ts

```diff
@@ -0,0 +1,29 @@
+import { useState } from "react";
+
+export default function useLocalStorage(key, initialValue) {
+  const [storedValue, setStoredValue] = useState(() => {
```

Comment:
```suggestion
  const [storedValue, setStoredValue] = React.useState(() => {
```

Reply:
It's also probably worth typing this function as well in the next series of PRs.

Reply:
Added allowSyntheticDefaultImports, see above.

Reply:
I'll make it * as React, and then React.functions